src/sections/Home/Playground-home/index.js

```diff
@@ -105,14 +111,20 @@ const ViewsSectionWrapper = styled.div`
 
     }
 
-    .overlay {
-        width: 483px;
-        height: 680px;
-    }
+    // .overlay {
```

Comment:
Sorry, I think when I keep updating the branch to get it on par with the main branch something keeps overriding that part because I’ve removed the commented code a few times now and it keeps popping back up lol